src/ducks/Signals/TriggerForm.module.scss

```diff
@@ -107,6 +107,14 @@ label {
 }
 
 .AboutForm {
+  box-sizing: border-box;
+  background-color: var(--white);
+  color: var(--black);
+  border-radius: 4px;
+  border: 1px solid var(--grey-light-3);
```

Comment:
`--grey-light-3` is a deprecated css variable. Please, use variables that are listed in our [design system](https://www.figma.com/file/a7RcFp4wWVCBl744AUmYykRF/Pattern-library?node-id=284%3A2)

Reply:
Will fix